### PR TITLE
contrib/micro: new package (2.0.13)

### PR DESCRIPTION
**Pull request by Bridouz**

### CODE REVIEW

File: contrib/micro/template.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+pkgname = "micro"
+pkgver = "2.0.13"
+pkgrel = 0
+build_style = "go"
+make_build_args = [
+    f"-ldflags=-X github.com/zyedidia/micro/v2/internal/util.Version={pkgver}",
+    "./cmd/micro",
+]
+hostmakedepends = ["go"]
+pkgdesc = "Modern and intuitive terminal-based text editor"
```
Comment (nekopsykose) [inline contrib/micro/template.py:10]:
```suggestion
pkgdesc = "Terminal-based text editor"
```

**Comment by nekopsykose** [comment]
you have to fix the ci lint too

**Comment by Bridouz** [comment]
> you have to fix the ci lint too

No info about it and I don't see what's wrong.

**Comment by nekopsykose** [comment]
but it is visible to you, the checks at the bottom of the page and in the checks tab at the top show it

**Comment by nekopsykose** [comment]
(you have to run `black` on the template)

**Comment by Bridouz** [comment]
Done ;)

**Comment by q66** [comment]
i think this looks good now, thanks

**Pull request merged**